internal/node_launcher.sh

```diff
@@ -46,16 +46,16 @@ esac
 
 if [[ -n "$TEST_SRCDIR" ]]; then
   # Case 4, bazel has identified runfiles for us.
```

Comment:
the docs talk about finding our runfiles tree, but this is finding our manifest. Update?

Reply:
done